src/fragments/lib/in-app-messaging/integrate-your-application/integrate-your-application.mdx

```diff
@@ -8,29 +8,25 @@ import js0 from '/src/fragments/lib/in-app-messaging/integrate-your-application/
 
 <Fragments fragments={{ js: js0 }} />
 
-### Configure Amplify
+import reactnative1 from '/src/fragments/lib/in-app-messaging/integrate-your-application/react-native/configure-amplify.mdx';
 
-Next, you should import and load the `aws-exports.js` configuration file created by the CLI. Add the Amplify configuration step to your application’s root entry point. For example, `index.js`.
-
-import reactnative3 from '/src/fragments/lib/in-app-messaging/integrate-your-application/react-native/configure-amplify.mdx';
-
-<Fragments fragments={{ 'react-native': reactnative3 }} />
+<Fragments fragments={{ 'react-native': reactnative1 }} />
 
-import js3 from '/src/fragments/lib/in-app-messaging/integrate-your-application/js/configure-amplify.mdx';
+import js1 from '/src/fragments/lib/in-app-messaging/integrate-your-application/js/configure-amplify.mdx';
```

Comment:
For future reference - we don't need to keep using the naming convention of `import js#` or `reactnative#`. Things in the docs are like that due to an automated migration in the past and if we can use something human-decipherable going forward, it would probably be more helpful for readers to figure out what fragments are going where